PayPalMessaging/src/main/java/com/braintreepayments/api/PayPalMessagingRequest.kt

```diff
@@ -17,10 +17,10 @@ package com.braintreepayments.api
  */
 data class PayPalMessagingRequest(
     var amount: Double? = null,
```

Comment:
random q: Is amount truly optional here? If yes, would a value like `0.0` work better? If no, I think it makes sense to remove the optional and require merchants to explicitly specify an amount.

Reply:
I'm not sure, @jaxdesmarais. Could you help me clarify this, please? Happy to make any necessary changes to this :D

Reply:
The amount is optional and is displayed on the popup if provided, so we won't want to default to `0.0` or make it required. We are defaulting to `null` here to match the [underlying SDK](https://github.com/paypal/paypal-messages-android/blob/d3293c07ed758724700d48642f1159e576c748e3/library/src/main/java/com/paypal/messages/config/message/PayPalMessageData.kt#L17) default.